.github/workflows/integrationTest.yml

```diff
@@ -138,6 +86,40 @@ jobs:
       - name: Upload to s3
         if: steps.cached_binaries.outputs.cache-hit != 'true'
         run: aws s3 cp build/bin s3://${S3_INTEGRATION_BUCKET}/integration-test/binary/${{ github.sha }} --recursive
+
+      - name: Login ECR
+        if: steps.cached_binaries.outputs.cache-hit != 'true'
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
+
+      - name: Set up Docker Buildx
+        if: steps.cached_binaries.outputs.cache-hit != 'true'
+        uses: docker/setup-buildx-action@v1
+
+      - name: Set up QEMU
+        if: steps.cached_binaries.outputs.cache-hit != 'true'
+        uses: docker/setup-qemu-action@v1
+
+      # build dir is ignored in our .dockerignore thus need to copy to another dir
+      - name: Copy Binary For Agent Image Build
+        if: steps.cached_binaries.outputs.cache-hit != 'true'
+        run: cp -r build/bin/linux/* .
+
+        #Build the cloudwatch agent image for two primary reasons:
+        #-Using the cloudwatch agent image to do the integration test (can be used for internal)
+        #-Export it for the future use in CD release pipeline
+        #Documentation: https://github.com/docker/build-push-action
+      - name: Build Cloudwatch Agent Image
+        uses: docker/build-push-action@v2
+        if: steps.cached_binaries.outputs.cache-hit != 'true'
+        with:
+          file: amazon-cloudwatch-container-insights/cloudwatch-agent-dockerfile/localdeb/Dockerfile
+          context: .
+          push: true
+          tags: |
+            ${{ steps.login-ecr.outputs.registry }}/${{ env.ECR_INTEGRATION_TEST_REPO }}:${{ github.sha }}
```

Comment:
Is ECR_INTEGRATION_TEST_REPO already documented in our integ test setup README?

Reply:
@khanhntd please comment as you wrote this. I just changed where this happens in build so I can copy the deb.

Reply:
I have not documented `ECR_INTEGRATION_TEST_REPO` in the README. Will take an action on this.